src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String startUp = " Hello! I'm Duke\n"
+                + " What can I do for you?";
+        ArrayList<Task> list = new ArrayList<>();
+        Scanner myObj = new Scanner(System.in);
+
+        msgTemplate(startUp);
+
+        while(true) {
+            String input = myObj.nextLine();
+            if (input.startsWith("todo")) {
+                try {
+                    if (input.charAt(4) != ' ') throw new IllegalArgumentException();
+                    if (input.equals("todo ")) throw new StringIndexOutOfBoundsException();
+                    String item = input.substring(5);
+                    list.add(new Todo(item));
+                    msgTemplate(" Got it. I've added this task:\n" + "  " + list.get(list.size() - 1) + System.lineSeparator() + " Now you have " + list.size() + " tasks in the list.");
+                } catch (StringIndexOutOfBoundsException e) {
+                    msgTemplate(" ☹ OOPS!!! The description of a todo cannot be empty.");
+                } catch (IllegalArgumentException e) {
+                    msgTemplate(" ☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                }
+            } else if (input.startsWith("deadline")) {
+                try {
+                    if (input.charAt(8) != ' ') throw new IllegalArgumentException();
```

Comment:
Maybe you can consider refactoring "deadline" into an enum, such that you can avoid hardcoding 8 over here, similar to the other commands

Reply:
Thanks for all the comments and suggestions (including the ones above)! Will keep these in mind! :)